cypress/integration/homepage-spec.js

```diff
@@ -0,0 +1,37 @@
+describe('Test home page', () => {
+  beforeEach(() => {
+    cy.visit('http://localhost:3000/');
+  });
+
+  it('Test text', () => {
+    cy.contains('TARGET MVD');
+    cy.contains('Find people near you & connect');
+    cy.contains(
+      'Create a target wherever on the map, specify your interest: Travel, Dating, Music, etc and start conecting with others who share your interest.'
+    );
+    cy.contains('EMAIL');
+    cy.contains('PASSWORD');
+    cy.contains('Forgot your password?');
+    cy.contains('CONNECT WITH FACEBOOK');
+    cy.contains('SIGN UP');
```

Comment:
don't you have these strings defined somewhere?

Reply:
Let me see if I can change this and put the `intl` function

Reply:
What is the goal of this test, to test that the components with those texts are displayed or that the texts themselves are correct?

If you reuse the string that is being used in the actual code here, an error where someone does a copy change that was not intentional wouldn't be caught.

But as I said, that might not be the goal of the test and reusing could be fine.

Reply:
I'm testing that the text is in the page. Is that ok>

Reply:
Well, there must be many opinions about this matter. What I can say is that to me it depends on the case, there are some cases where the text is not that important, especially long texts that are more prone to change. But there are others, like texts in buttons and such, that it might be a good idea to decouple from the app locale. I'd say to play it by ear by gaging the importance of the text in question.